code/modules/food_and_drink/snacks.dm

```diff
@@ -128,8 +139,50 @@
 				user.visible_message("<span class='alert'><b>[user]</b> futilely attempts to shove [src] into [M]'s mouth!</span>")
 				return
 		else
+			if (sharpened)
+				boutput(M, "<span class='alert'>The pizza was too pointy!</span>")
```

Comment:
both this and the above pointy hurt can both occur, why??

Reply:
Eating the pizza hurts a lot, but trying to feed it to someone also has a chance of causing bleeding.